skycultures/indian/constellation_names.eng.fab

```diff
@@ -19,49 +20,85 @@
+# For column 3: We've referred to highly regarded sanskrit to sanskrit dictionaries kalpadruma and vAchaspatyam for etymologies below.
 
-# We use IAST transliteration for unambious transliteration of devanAgarI using latin alphabets, while using phonetically more intuitive equivalents in suggested translations.
+#################
+# Nakshatra-s.
+#################
+# We show Nakṣatra-s associated with a single star as a star, rather than as a constellation.
+# From @gzotti: "if some Nakshatras may receive sky art, all Nakshatras should be coded as constellations. When we do the change mentioned above, those will then be separated (or just marked inside the constellation file). ... We may highlight the 1-star constellation in later editions e.g. by a circle." 
 
-## We've referred to highly regarded sanskrit to sanskrit dictionaries kalpadruma and vAchaspatyam for etymologies below.
+Aśv	"Aśvinī"	_("Divine physician twins")
+Bhar	"Bharanī"	_("Bearer")
+Kṛt	"Kṛttikā"	_("Cutter")
+Roh	"Rohiṇī"	_("Red")
+Ārd	"Ārdrā"	_("Wet")
+PVas	"Punarvasū"	_("Wealth-restorer")
+Puṣ	"Puṣya"	_("Nourisher")
+Magh	"Maghā"	_("Bountiful")
+# E. फल् to bear fruit, Unādi aff. उनन् and गुक् aug.
+PPhā	"Pūrva phālgunī"	_("Former Fruitful")
+UPhā	"Uttara phālgunī"	_("Latter Fruitful")
+Viś	"Viśākhā"	_("Well branched")
+Anur	"Anurādhā"	_("Following Rādhā")
+Jyeṣ	"Jyeṣṭhā"	_("Elder")
+PAṣ	"Pūrva aṣāḍhā"	_("Former invincible one")
+UAṣ	"Uttara aṣāḍhā"	_("Latter invincible one")
+Dhani	"Dhaniṣṭhā"	_("Wealthiest")
+# Provenance for art - BG Tilak's book (The Orion; or, Researches into the Antiquity of the Vedas), veda-s (eg. Aitareya AraNyaka quoted elsewhere in this doc). 
+Mṛg	"Mṛgaśiras"	_("Deer's head")
+# Associated with the snakes, as per taittirIya bhrAhmaNa.
+Aśl	"Aśleṣa"	_("Embrace")
+Hasta	"Hasta"	_("Hand")
+Mūl	"Mūla"	_("Root")
+Śra	"Śravaṇa"	_("Ear")
+PBhā	"Pūrva bhādrapada"	_("The first of the blessed feet")
+UBhā	"Uttara bhādrapada"	_("The second of the blessed feet")
+Chitr	"Chitrā"	_("Spectacular")
+Svā	"Svātī"	_("Svātī")	
+Śata	"Śatabhiṣak"	_("Hundred cures")	
+Rev	"Revatī"	_("The prosperous one")	
+Abh	"Abhijit"	_("The invincible one")
 
-# TRANSLATORS: Horse headed twin son of The Sun
-N01	"Aśvinī"		_("Twin physicians of Gods")
-# TRANSLATORS: Bearer of life
-N02	"Bharanī"		_("She who bears")
-# TRANSLATORS: kartiikeya - Son of Lord Shiva
-N03	"Kṛittikā"		_("Nurses of kartiikeya")
-# TRANSLATORS: Deer avatar of Uṣa or Dwan
-N04	"Rohiṇī"		_("The spirited one")
-# TRANSLATORS: Deer avatar of King Prajapati
-N05	"Mṛgaśirā"		_("The deer's head")
-# TRANSLATORS: Rudra, Lord Shiva
-N06	"Ārdrā"			_("The moist one")
-N07	"Punarvasu"		_("The restorers of goods")
-N08	"Puṣyā"			_("The nourisher")
-N09	"Aśleṣā"		_("The embrace")
-N10	"Maghā"			_("The bountiful one")
-# TRANSLATORS: phālgun - Red
-N11	"Pūrba phālgunī"	_("The former red one")
```

Comment:
Thanks for updating this and other entries with a standardised spelling (Pūrba -> Pūrva).